code/modules/atmospherics/machinery/datum_pipeline.dm

```diff
@@ -95,6 +95,7 @@
 						pipenetwarnings--
 					if(pipenetwarnings == 0)
 						log_mapping("build_pipeline(): further messages about pipenets will be suppressed")
+						pipenetwarnings--
```

Comment:
```suggestion
					if(pipenetwarnings == 0)
						log_mapping("build_pipeline(): further messages about pipenets will be suppressed")
					pipenetwarnings--
```

Reply:
it wouldn't overflow regardless. Byond vars don't over or underflow. The systems we use to interact with those numbers can break, or in some situations cause overflow like behavior but they won't do it on their own with just + and -

Reply:
done.